src/db.rs

```diff
@@ -1152,6 +1152,48 @@ impl DB {
             ))
         })
     }
+
+    pub fn property_value(&self, name: &str) -> Option<String> {
+        let prop_name = CString::new(name).unwrap();
+        unsafe {
+            let value = ffi::rocksdb_property_value(self.inner, prop_name.as_ptr());
+            if value.is_null() {
+                return None;
+            }
+
+            let s = CStr::from_ptr(value).to_str().unwrap().to_owned();
```

Comment:
I propose to avoid `unwrap()` where it's possible. 

```suggestion
            let s = CStr::from_ptr(value).to_str()?.to_owned();
```